config/locales/ar.yml

```diff
@@ -794,16 +794,16 @@ ar:
     policies: سياسات
   worldwide_organisation:
     corporate_information:
-      about_our_services_html:
-      personal_information_charter_html:
-      publication_scheme_html:
-      social_media_use_html:
-      welsh_language_scheme_html:
-    find_out_more:
+      about_our_services_html: تعرف على %{link}.
+      personal_information_charter_html: يوضح %{link} كيف نتعامل مع معلوماتك الشخصية.
+      publication_scheme_html: اقرأ عن أنواع المعلومات التي ننشرها بشكل روتيني في %{link}.
+      social_media_use_html: اقرأ سياستنا على %{link}.
+      welsh_language_scheme_html: تعرف على التزامنا بشأن %{link}.
+    find_out_more: اطلع على الملف الكامل وجميع تفاصيل الاتصال
     headings:
-      contact_us:
-      corporate_information:
-      follow_us:
-      our_people:
-    location:
-    part_of:
+      contact_us: اتصل بنا
+      corporate_information: معلومات الشركة
+      follow_us: تابعنا
+      our_people: موظفونا
+    location: الموقع
+    part_of: جزء من
```

Comment:
Could we squash this commit with the other translations commit?

Minor - missing a newline here and also at the bottom of be.yml. Think those are the only ones but worth having a check